src/collectors/python.d.plugin/hpssa/hpssa.chart.py

```diff
@@ -86,7 +86,7 @@
         Physical[ ]Drives
         | None[ ]attached
         | (?:Expander|Enclosure|SEP|Port[ ]Name:)[ ].+
-        | .+[ ]at[ ]Port[ ]\S+,[ ]Box[ ]\d+,[ ].+
+        | .+[ ]at[ ]Port[ ]\S+,[ ]Box[ ]\d+(\s\(Index[ ]\d+\))?,[ ].+
```

Comment:
We could also relax the regex like
```suggestion
        | .+[ ]at[ ]Port[ ]\S+,[ ]Box[ ]\d+
```

I am not sure we need to be super specific here.